### ISSUE TITLE
hideous amounts of css lag for some reason

### ISSUE THREAD

**Issue by toydotgame** [milbits/oldcord#45]
hey gamer,,the last couple of days (ive updated from months old 32 bit discord to brand-new-as-of-right-now discord) theres been massive amounts of lag literally only when i have oldcord loaded as a theme. very occasionally in some areas things go smooth but a good 90% of the time typing moving the mouse around clicking on stuff is like a good 3 fps tops

it doesnt seem to be dependent on it being a server, dm, nor whatever is in the actual chat causing lag, it just randomly does this. i have all three of ur oldcord modification css but without them its still insane

im on kde 5 arch linux btw but i literally have no idea how to diagnose css lag so this is kinda a low importance draft issue rn, im suffering but theres not much i can do to figure it out,,im letting you know to see if this can be recreated but if not its so joever lmao oh well

**Comment by toydotgame** [comment]
yeah i'm running amdgpu so i have no idea. but i am running vencord+openasar and my packages were last updated in april so there might be something there

**Comment by toydotgame** [comment]
does the hwaccel option in just normal discord change anything? probably not at this point if gpu isnt the cause. i dont even think chromium css rendering _can_ use hwaccel at all. no idea where it comes from :((

**Comment by toydotgame** [comment]
tested on win10 ltsc vesktop 32 bit w/o hwaccell (8 gb ram i5-8350U and som fuckass ssd)
![image](https://github.com/milbits/oldcord/assets/64190512/f36cee48-98c2-45b5-9b52-fd8dcd48a604)
![image](https://github.com/milbits/oldcord/assets/64190512/3e23f9bb-424a-48d6-a507-261f01cd8644)

still horrendous </3

literally how do you diagnose something that is just css. the hover effects on my website are also css only and they lag a bit so yeah css is laggy but like???how

**Comment by TorutheRedFox** [comment]
still a problem sadly :c

**Comment by toydotgame** [comment]
css try to not be comically fuckign laggy challenge EPIC FAIL

**Comment by KingGamingYT** [comment]
odd

legitimately the only reason I can think that this happens is because the theme is split into multiple files and relies on imports, which isn't the norm
but those files are stored locally when imported and technically have no explicit reason to be connected, they're just intended to be
idk

**Comment by KingGamingYT** [comment]
curse the canadian goose

**Comment by KingGamingYT** [comment]
![image](https://github.com/user-attachments/assets/dc510872-6d1a-44f4-b6f2-7ff44abe15d6)
surely can't imagine why the theme has lag when we rely on selectors like this (I still don't get lag)

**Comment by KingGamingYT** [comment]
very odd, legitimately the only major lag I've experienced is with dav's icon pack enabled

**Comment by milbits** [comment]
i think i fixed this almost completely, only the profile modal is stuttering
